From 3d1a0fee5489d310e9c31b0d8f345bd149e3bcd9 Mon Sep 17 00:00:00 2001
From: Chirayu Desai <chirayudesai1@gmail.com>
Date: Tue, 7 Aug 2012 19:19:33 +0530
Subject: [PATCH 1/2] Dev Tools: hide the icon!

change default catergory of the main activity to DEFAULT
instead of LAUNCHER

Change-Id: I25ed302ea3eedb93b5119a83f08b6d50aaecce08
---
 apps/Development/AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apps/Development/AndroidManifest.xml b/apps/Development/AndroidManifest.xml
index 2379e00..f09434c 100644
--- a/apps/Development/AndroidManifest.xml
+++ b/apps/Development/AndroidManifest.xml
@@ -56,7 +56,7 @@
             android:icon="@mipmap/ic_launcher_devtools">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-                <category android:name="android.intent.category.LAUNCHER" />
+                <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
         <activity android:name="PackageBrowser" android:label="Package Browser">
-- 
2.9.3

